src/Microsoft.Diagnostics.Monitoring.Options/OptionsDisplayStrings.resx

```diff
@@ -125,6 +125,62 @@
     <value>Hash algorithm used to compute ApiKeyHash, typically 'SHA256'. 'SHA1' and 'MD5' are not allowed.</value>
     <comment>The description provided for the ApiKeyHashType parameter on ApiAuthenticationOptions.</comment>
   </data>
+  <data name="DisplayAttributeDescription_AspNetRequestCountOptions_ExcludePaths" xml:space="preserve">
+    <value>The list of request paths to ignore.</value>
+    <comment>The description provided for the ExcludePaths parameter on AspNetRequestCountOptions.</comment>
+  </data>
+  <data name="DisplayAttributeDescription_AspNetRequestCountOptions_IncludePaths" xml:space="preserve">
```

Comment:
Does this support wild carding like `/path/to/*` or does `/path/to/` match all sub paths?, it might be worth it to make this callout in the description.

Reply:
The spec currently does not call this out either way.

The initial implementation should match the path exactly (give or take the trailing slash). Then we can add wildcards later.

We'll need to discuss the wildcard system before that's implemented or a library is pulled in. I'd like to see wildcards, but we'll want to keep it relatively simple rather going full regex.

Reply:
I'll update the description to say (for now) that the path must match exactly.